client/js/socket-events/auth.js

```diff
@@ -3,15 +3,34 @@
 const $ = require("jquery");
 const socket = require("../socket");
 const storage = require("../localStorage");
+const utils = require("../utils");
 
 socket.on("auth", function(data) {
+	// If we reconnected and serverHash differs, that means the server restarted
+	// And we will reload the page to grab the latest version
+	if (utils.serverHash > -1 && data.serverHash > -1 && data.serverHash !== utils.serverHash) {
+		socket.disconnect();
+		$("#connection-error").text("Reloading the page…");
```

Comment:
Nitpick but just `Reloading…` or `Refreshing…` is enough IMO.

Reply:
What about `Server restarted, reloading…` as it's better to explain why it's reloading.

Reply:
Works for me